responses/sl/HassGetState.yaml

```diff
@@ -49,6 +61,32 @@ responses:
           {%- endfor %}
         {% endif %}
 
+      everybody: |
+        {% if not query.unmatched: %}
+          Da
+        {% else %}
+          {% set no_match = query.unmatched | map(attribute="name") | sort | list %}
+          {% if no_match | length > 4 %}
+            Ne, {{ no_match[:3] | join(", ") }} in {{ (no_match | length - 3) }} niso
+            {% if state.attributes.get("untranslated_state") == "not_home" %}
+              {{ state.state }}
+            {% else %}
+              v {{ state.state }}
+            {% endif %}
+          {%- else -%}
+            Ne,
+            {% for name in no_match -%}
+              {% if not loop.first and not loop.last %}, {% elif loop.last and not loop.first %} in {% endif -%}
+              {{ name }}
+            {%- endfor %} {% if no_match | length > 1 %}in{% else %}is{% endif %} ni
+            {% if state.attributes.get("untranslated_state") == "not_home" %}
+              {{ state.state }}
+            {% else %}
+               {{ state.state }}
+            {% endif %}
```

Comment:
Same here, there's no need for the if

Reply:
Hm, yes, there was "at" which doesn't sound right if I just put "v" (as stands in Slovenian for at).
`John is away`- > `John je proč`
`John is at work` -> `John je v službi`

Reply:
https://github.com/home-assistant/intents/pull/1769/files#diff-05a49728660604fda371e824b9dfcd1487b062f2e9f804130b8460a9a3ddaa14R23

Same in Romanian, here's the fix